server/test/uk/gov/ons/addressIndex/server/model/response/AddressResponseAddressSpec.scala

```diff
@@ -75,11 +76,230 @@ class AddressResponseAddressSpec extends WordSpec with Matchers {
       )
 
       // When
-      val result = AddressResponseAddress.fromPafAddress(given)
+      val result = AddressResponseAddress.fromPafAddress(givenPaf)
+
+      // Then
+      result shouldBe expected
+    }
+
+    "be creatable from Elastic NAG response" in {
+      // Given
+      val givenNag = NationalAddressGazetteerAddress(
+        uprn = "n1",
+        postcodeLocator = "n2",
+        addressBasePostal = "n3",
+        ursn = "n4",
+        lpiKey = "n5",
+        paoText = "n6",
+        paoStartNumber = "n7",
+        paoStartSuffix = "n8",
+        paoEndNumber = "n9",
+        paoEndSuffix = "n10",
+        saoText = "n11",
+        saoStartNumber = "n12",
+        saoStartSuffix = "n13",
+        saoEndNumber = "n14",
+        saoEndSuffix = "n15",
+        level = "n16",
+        officialFlag = "n17",
+        logicalStatus = "n18",
+        streetDescriptor = "n19",
+        townName = "n20",
+        locality = "n21",
+        organisation = "n22",
+        legalName = "n23",
+        latitude = "1.0000000",
+        longitude = "2.0000000",
+        score = 1.0f
+      )
+
+      val expected = AddressResponseAddress(
+        uprn = "n1",
+        formattedAddress = "",
+        paf = None,
+        nag = Some(AddressResponseNag(
+          uprn = "n1",
+          postcodeLocator = "n2",
+          addressBasePostal = "n3",
+          ursn = "n4",
+          lpiKey = "n5",
+          paoText = "n6",
+          paoStartNumber = "n7",
+          paoStartSuffix = "n8",
+          paoEndNumber = "n9",
+          paoEndSuffix = "n10",
+          saoText = "n11",
+          saoStartNumber = "n12",
+          saoStartSuffix = "n13",
+          saoEndNumber = "n14",
+          saoEndSuffix = "n15",
+          level = "n16",
+          logicalStatus = "n18",
+          streetDescriptor = "n19",
+          townName = "n20",
+          locality = "n21",
+          organisation = "n22",
+          legalName = "n23"
+        )),
+        geo = Some(AddressResponseGeo(
+          latitude = 1.0d,
+          longitude = 2.0d,
+          easting = 0,
+          northing = 0
+        )),
+        underlyingScore = 1,
+        underlyingMaxScore = 1
+      )
+
+      // When
+      val result = AddressResponseAddress.fromNagAddress(1)(givenNag)
+
+      // Then
+      result shouldBe expected
+    }
+
+    "be creatable from Elastic PAF response with wrong latitude" in {
```

Comment:
"be creatable" Is not a start of a sentence, hence replace the test name.
Something more descriptive of whats going on here.